packages/editor/src/store/test/selectors.js

```diff
@@ -2766,228 +2733,6 @@ describe( 'selectors', () => {
 		} );
 	} );
 
-	describe( '__experimentalGetDefaultTemplateTypes', () => {
-		const state = { editorSettings: { defaultTemplateTypes } };
-
-		it( 'returns undefined if there are no default template types', () => {
-			const emptyState = { editorSettings: {} };
-			expect(
-				__experimentalGetDefaultTemplateTypes( emptyState )
-			).toBeUndefined();
-		} );
-
-		it( 'returns a list of default template types if present in state', () => {
-			expect(
-				__experimentalGetDefaultTemplateTypes( state )
-			).toHaveLength( 2 );
-		} );
-	} );
-
-	describe( '__experimentalGetDefaultTemplatePartAreas', () => {
-		const state = { editorSettings: { defaultTemplatePartAreas } };
-
-		it( 'returns empty array if there are no default template part areas', () => {
-			const emptyState = { editorSettings: {} };
-			expect(
-				__experimentalGetDefaultTemplatePartAreas( emptyState )
-			).toHaveLength( 0 );
-		} );
-
-		it( 'returns a list of default template part areas if present in state', () => {
-			expect(
-				__experimentalGetDefaultTemplatePartAreas( state )
-			).toHaveLength( 2 );
-		} );
-
-		it( 'assigns an icon to each area', () => {
-			const templatePartAreas =
-				__experimentalGetDefaultTemplatePartAreas( state );
-			templatePartAreas.forEach( ( area ) =>
-				expect( area.icon ).not.toBeNull()
-			);
-		} );
-	} );
-
-	describe( '__experimentalGetDefaultTemplateType', () => {
-		const state = { editorSettings: { defaultTemplateTypes } };
-
-		it( 'returns an empty object if there are no default template types', () => {
-			const emptyState = { editorSettings: {} };
-			expect(
-				__experimentalGetDefaultTemplateType( emptyState, 'slug' )
-			).toEqual( {} );
-		} );
-
-		it( 'returns an empty object if the requested slug is not found', () => {
-			expect(
-				__experimentalGetDefaultTemplateType( state, 'foobar' )
-			).toEqual( {} );
-		} );
-
-		it( 'returns the requested default template type', () => {
-			expect(
-				__experimentalGetDefaultTemplateType( state, 'index' )
-			).toEqual( {
-				title: 'Default (Index)',
-				description: 'Main template',
-				slug: 'index',
-			} );
-		} );
-
-		it( 'returns the requested default template type even when the slug is numeric', () => {
-			expect(
-				__experimentalGetDefaultTemplateType( state, '404' )
-			).toEqual( {
-				title: '404 (Not Found)',
-				description: 'Applied when content cannot be found',
-				slug: '404',
-			} );
-		} );
-	} );
-
-	describe( '__experimentalGetTemplateInfo', () => {
```

Comment:
I created the same unit tests for the `getTemplateInfo` function: 2a04ba14fbe2bb575cbe53aaad63a2809c1c3408